index.html

```diff
@@ -31,6 +31,148 @@ <h2 class="position">Software Engineer</h2>
           you need coded, don’t hestiate to contact me.
         </p>
       </section>
+
+      <section class="work">
+        <h1>MY RECENT WORK</h1>
+        <hr />
+
+        <div class="project-grid">
+          <div class="project project1">
+            <img src="images/project.png" alt="project2" />
+            <div class="project-card">
+              <h1 class="project-title">Multi-Post Stories</h1>
+              <p class="project-descrption">
+                A daily selection of privately personalized reads; no accounts
+                or sign-ups required. has been the industry's standard dummy
+                text ever since the 1500s, when an unknown printer took a
+                standard dummy text.
+              </p>
+              <div class="tech-stack">
+                <div class="tech1">css</div>
+                <div class="tech2">html</div>
+                <div class="tech3">bootstrap</div>
+                <div class="tech4">Ruby</div>
+              </div>
```

Comment:
- [ ] Kindly  ensure that the list of project technologies are a list `ul` and each of them a `li` tag as required by the previous reviewer😉

Reply:
- hi @diwakarstha, 
Nice job for implementing these changes. :+1: